public/global.css

```diff
@@ -1,63 +1,61 @@
-html, body {
-	position: relative;
-	width: 100%;
-	height: 100%;
+:root {
+  --font-regular: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto,
+    Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif;
+  --font-monospace: "Courier New", Courier, monospace;
 }
 
-body {
-	color: #333;
-	margin: 0;
-	padding: 8px;
-	box-sizing: border-box;
-	font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif;
-}
-
-a {
-	color: rgb(0,100,200);
-	text-decoration: none;
+/* See https://www.30secondsofcode.org/css/s/box-sizing-reset */
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
 }
 
-a:hover {
```

Comment:
I got rid of all the CSS rules associated with `<a>` elements because there are no `<a>` elements in your markup. The worst thing you can do with CSS is to just have a bunch of crap that never gets used anywhere. I think that's why people tend to hate CSS.

Reply:
Gotcha